about.html

```diff
@@ -0,0 +1,223 @@
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+          <a class="text-decoration-none me-4" href="">Log Out</a>
+          <a class="text-decoration-none me-4" href="">My Page</a>
+          <a class="text-decoration-none me-4" href="">Burmese</a>
+        </div>
+        <section class="wrap pb-5">
+          <header class="menu-wrap">
+            <nav>
+              <input
+                type="checkbox"
+                class="toggler"
+                id="toggle"
+                onclick="toggle()"
+              />
+              <div class="hamburger"><div></div></div>
+              <div class="menu">
+                <div class="overlay">
+                  <div>
+                    <ul class="menubar">
+                      <li>
+                        <a href="index.html" class="brand">
+                          <img src="./img/logo.png" alt="logo" />
+                        </a>
+                      </li>
+                      <li>
+                        <ul class="nav-wrap">
+                          <li>
+                            <a href="about.html" class="nav-links">About</a>
+                          </li>
+                          <li><a href="#" class="nav-links">Program</a></li>
+                          <li>
+                            <a href="#works" class="nav-links">Buy Tickets</a>
+                          </li>
+                          <li>
+                            <a href="#works" class="nav-links">Sponsor</a>
+                          </li>
+                          <li><a href="#works" class="nav-links">News</a></li>
+                          <li class="cc">
+                            <a href="index.html" class="textcc">
+                              Fundraising Concert
+                            </a>
+                          </li>
+                        </ul>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </nav>
+          </header>
+          <div class="intro about-wrap">
+            <p id="introText" class="hide">"Hello! Sharing World"</p>
+            <h1 class="heading mb-4 me-3">
+              Fund Raising
+              <br />
+              Concert in Myanmar 2023
+            </h1>
+          </div>
+          <div class="mb-lg-5 detail-wrap about-wrap">
+            <p class="detail detail-about">
+              The concert brings together the community of activists, experts
+              and all the people around the world. This year, we hope to expand
+              our invitation list including organizations and individuals
+            </p>
+            <p class="contact contact-about">
+              Please contact us per Email for any further questions about the
+              Fund Raising concert 2023!
+              <br />
+              <a href="#" class="fundLink">fundraising2023@icmyanmar.org</a>
+            </p>
+          </div>
```

Comment:
- [ ] For proper page structure, this section should be in a separate section by itself and not nested inside the same section s the header (meaning it should not be in the same section as the header),